mono/profiler/Makefile.am

```diff
@@ -85,8 +85,17 @@ else
 if HOST_TIZEN
 libmono_dep = $(top_builddir)/mono/mini/$(LIBMONO_LA)
 else
+# BITCODE => -no-undefined
```

Comment:
This change is already part of https://github.com/mono/mono/pull/8969, please remove

Reply:
Right. I needed something that works more, will be fixed by merging.

Reply:
Llvm-dis “needed” on regular Sierra. Maybe not on HighSierra. Could also autoconf but ok if 0, once I confirm highsierra works for me.